Realm.Shared/handles/SchemaHandle.cs

```diff
@@ -0,0 +1,88 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Realms
+{
+    internal class SchemaHandle : RealmHandle
+    {
+        private static class NativeMethods
+        {
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "schema_create", CallingConvention = CallingConvention.Cdecl)]
+            public static extern unsafe IntPtr create([MarshalAs(UnmanagedType.LPArray), In] Native.SchemaObject[] objects, int objects_length,
+                                                              [MarshalAs(UnmanagedType.LPArray), In] Native.SchemaProperty[] properties,
```

Comment:
Scratch my indentation comment - as @fealebenpae said, suspect this entire file snuck back in during a merge.